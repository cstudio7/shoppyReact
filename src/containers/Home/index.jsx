import React from 'react';
import Navbar from '../Navbar';
import DashboardCard from '../../components/DashboardCard';

export default function Home() {
  return <Navbar displayPage={<DashboardCard />} />;
}
