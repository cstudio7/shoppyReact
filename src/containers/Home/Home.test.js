import React from 'react';
import { shallow } from 'enzyme';
import Home from './index';

describe('Test the Home Component', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<Home />);
    expect(wrapper).toMatchSnapshot();
  });
});
