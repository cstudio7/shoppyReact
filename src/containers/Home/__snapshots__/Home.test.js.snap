// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test the AdminNav Component should match snapshot 1`] = `
<Navbar
  displayPage={<InfoCard />}
/>
`;
